esx_service/vmdk_ops.py

```diff
@@ -1183,6 +1188,7 @@ def setStatusAttached(vmdk_path, vm, vm_dev_info=None):
     if not vol_meta:
         vol_meta = {}
     vol_meta[kv.STATUS] = kv.ATTACHED
+    # instanceUuid i.e. vc uuid of a VM is stored in volume kv
```

Comment:
instanceUuid doesn't exist in vm.config object? So we need to save bios uuid (vm.config.uuid) in KV ?